RELEASE.md

```diff
@@ -0,0 +1,48 @@
+# Aptos Release Process
+
+## Branches and Tags
+
+```
+========================================= main branch ==========================================>
+                           \                                  \                         \
+                            \___aptos-node-v1.2.0 tag          \                         \
+                             \                                  \                         \
+                              \      aptos-framework-v1.3.0 tag__\                     devnet branch
+   aptos-framework-v1.2.0 tag__\                                  \                     
+                                \___aptos-node-v1.2.4 tag          \___aptos-node-v1.3.0 tag
+                                 \                                  \
+                                  \                                  \
+                             aptos-release-v1.2 branch         aptos-release-v1.3 branch
+
+```
+
+### main branch
+All current development occurs on the `main` branch. All new feature development
+should use feature flag to gate it off during development, and turn it on once the
+development is complete and passed the AIP process when it applies.
+
+### devnet branch
+The `devnet` branch is created of the `main` branch every week. It will be used to deploy
```

Comment:
Why don't we use tag here? Similar to how we can use mainnet/testnet/mainnet-framework/testnet-framework tags

Reply:
It would likely be very spammy in our release page if we start creating devnet tags for every devnet release.

Reply:
Also note that tags can't be moved very easily, if you created a tag on a specific commit, it's somewhat forever. So in github, arbitrary tag won't work, we can't just have a `devnet` tag, and move it whenever we cut new commit. every tag should be unique.

Reply:
I see